Calculation.py

```diff
@@ -0,0 +1,18 @@
+import output as op
+import spouseOrParentShares as sps
+
+def calculate(amount,sons,daughters,mother,father,wife):
+    if amount > 0:
```

Comment:
Docstrings are missing, what does this function do?
 Refer back to https://peps.python.org/pep-0008/